Documentation/ceph-quickstart.md

```diff
@@ -78,13 +78,16 @@ kubectl create -f operator.yaml
 kubectl -n rook-ceph get pod
 ```
 
-**NOTE**
-
-If you are using kubernetes v1.15 or older you need to create CRDs found here `/cluster/examples/kubernetes/ceph` as apiextension v1beta1 version of CustomResourceDefinition
-is deprecated in Kubernetes v1.16 and will no longer be supported from k8s v1.22.
-
 You can also deploy the operator with the [Rook Helm Chart](helm-operator.md).
 
+Before you start the operator in production, there are some settings that you may want to consider:
+1. If you are using kubernetes v1.15 or older you need to create CRDs found here `/cluster/examples/kubernetes/ceph/pre-k8s-1.16/crd.yaml`.
+   The apiextension v1beta1 version of CustomResourceDefinition was deprecated in Kubernetes v1.16.
+2. Consider if you want to enable certain Rook features that are disabled by default. See the [operator.yaml](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/operator.yaml) for these and other advanced settings.
+   1. Device discovery: In bare metal clusters, Rook will watch for new devices to configure if the `ROOK_ENABLE_DISCOVERY_DAEMON` setting is enabled.
```

Comment:
How about something like `In non-PVC clusters` or so?

Reply:
What about this? In the quick start guide, the user hasn't learned about the PVC vs non-PVC clusters.
Device discovery: Rook will watch for new devices to configure if the `ROOK_ENABLE_DISCOVERY_DAEMON` setting is enabled, commonly used in bare metal clusters.